test/test_ncbi_taxa.py

```diff
@@ -386,284 +695,316 @@ def test_id_lookup_errors(fixture_ncbi_validator, test_input, expected_exception
             dict(
                 nnme="E coli",
-                taxah={"genus": "Escherichia", "species": "Escherichia coli"},
+                taxon_hier=[("genus", "Escherichia"), ("species", "Escherichia coli")],
             ),
-            ("Escherichia coli", "species", 562, False, None, 561),
+            ("species", "Escherichia coli", 562, 561),
+            does_not_raise(),
+            (None, None),
+            ((INFO, "Match found for E coli"),),
+            id="Simple species",
         ),
-        (
+        pytest.param(
             dict(
                 nnme="Entero",
-                taxah={"order": "Enterobacterales", "family": "Enterobacteriaceae"},
+                taxon_hier=[
+                    ("order", "Enterobacterales"),
+                    ("family", "Enterobacteriaceae"),
+                ],
             ),
-            ("Enterobacteriaceae", "family", 543, False, None, 91347),
+            ("family", "Enterobacteriaceae", 543, 91347),
+            does_not_raise(),
+            (None, None),
+            ((INFO, "Match found for Entero"),),
+            id="Simple family",
         ),
-        (
+        pytest.param(
             dict(
                 nnme="E coli strain",
-                taxah={
-                    "species": "Escherichia coli",
-                    "strain": "Escherichia coli 1-110-08_S1_C1",
-                },
+                taxon_hier=[
+                    ("species", "Escherichia coli"),
+                    ("strain", "Escherichia coli 1-110-08_S1_C1"),
+                ],
             ),
-            ("Escherichia coli 1-110-08_S1_C1", "strain", 1444049, False, None, 562),
+            ("strain", "Escherichia coli 1-110-08_S1_C1", 1444049, 562),
+            does_not_raise(),
+            (None, None),
+            ((INFO, "Match found for E coli strain"),),
+            id="Simple strain",
         ),
-        (
+        pytest.param(
             dict(
                 nnme="Strepto",
-                taxah={"phylum": "Streptophyta", "subphylum": "Streptophytina"},
+                taxon_hier=[
+                    ("phylum", "Streptophyta"),
+                    ("subphylum", "Streptophytina"),
+                ],
             ),
-            ("Streptophytina", "subphylum", 131221, False, None, 35493),
-        ),
-        (
+            ("subphylum", "Streptophytina", 131221, 35493),
+            does_not_raise(),
+            (None, None),
+            ((INFO, "Match found for Strepto"),),
+            id="Simple phylum",
+        ),
+        # pytest.param(
+        #     dict(
+        #         nnme="Opistho",
+        #         taxon_hier=[("superkingdom", "Eukaryota"), ("clade", "Opisthokonta")],
+        #     ),
+        #     ("clade", "Opisthokonta", 33154, 2759),
+        #     does_not_raise(),
+        #     (None, None),
+        #     ((INFO, "Match found for Opistho"),),
+        #     id="Clade anchored by superkingdom",
+        # ),
```

Comment:
Guess this one should either be commented back in or deleted

Reply:
Yeah - I don't see how this can currently occur since we don't allow `clade` fields in the worksheet at present.

Reply:
Yeah I don't see why anyone would want to report a clade, so think test can go